solidity/contracts/converter/ConverterBase.sol

```diff
@@ -231,30 +213,32 @@ abstract contract ConverterBase is ConverterVersion, IConverter, TokenHolder, Co
     }
 
     /**
-     * @dev withdraws tokens held by the converter and sends them to an account
-     * can only be called by the owner
-     * note that reserve tokens can only be withdrawn by the owner while the converter is inactive
-     * unless the owner is the converter upgrader contract
+     * @dev transfers reserves to a new converter during an upgrade
+     * can only be called by the converter upgraded which should be set at its owner
      *
-     * @param _token   ERC20 token contract address
-     * @param _to      account to receive the new amount
-     * @param _amount  amount to withdraw
+     * @param _newConverter account to receive the new amount
```

Comment:
If it's a converter, then we should say 'address of the converter...'.

If it's some generic account, then we should use variable name 'account' or 'recipient' or similar.

Reply:
Sure

Reply:
👍